File: FreeYourFridge.API/DTOs/RecipeToList.cs
using FreeYourFridge.API.Models;
using Newtonsoft.Json;

namespace FreeYourFridge.API.DTOs
{
    public class RecipeToList
    {
        public int Id { get; set; }
        public string Image { get; set; }
        public string Title { get; set; }
        public Ingredient[] MissedIngredients { get; set; }
        [JsonProperty("usedIngredients")]
        public Ingredient[] UsedIngredients { get; set; }

    }
}

File: FreeYourFridge-SPA/src/app/_models/recipeToList.ts
import { MissedIngredients } from './missedIngredients';
import { UsedIngredients } from './usedIngredients';

export interface RecipeToList {
  id: number;
  title: string;
  image: string;
  missedIngredients: MissedIngredients[];
  usedIngredients: UsedIngredients[];
}
